docs/source/index.rst

```diff
@@ -80,13 +80,14 @@ Reminder - legacy PDFs are no longer being updated.
    Screening - Intro <./start-screening.rst>
    
 .. toctree::
-   :maxdepth: 1
+   :maxdepth: 2
```

Comment:
I test the documentation with '   :maxdepth: 2'   ;  I don't see any visual effect, probably because everything is level 1 under Resources.

Consider moving  'ResEvap Documentation' under Algorithms --> CWMS Only - Hydrologic